src/shorthands.jl

```diff
@@ -5,51 +5,48 @@ function ylims! end
 function zlims! end
 
 """
-    xlabel!([scene,] xlabel)
+    xlabel!([axis,] xlabel)
 
-Set the x-axis label for the given Scene.
-Defaults to using the current Scene.
+Set the x-axis label for the given Axis.
+Defaults to using the current Axis.
 """
-function xlabel!(scene, xlabel::AbstractString)
-    axis = scene[OldAxis]
```

Comment:
We might want to keep this implementation for people who are still using the old axes.  Perhaps you could have this implementation dispatch on the signature `xlabel!(scene::Scene, xlabel::AbstractString)`?

Reply:
I see - I will keep the scene implementation in `shorthands.jl`. 

I'm a bit confused though - why can I refer to dispatching `::Scene` in `shorthands.jl` but not `::Axis`? Neither is defined in the scope of this file. What am I missing here?

Reply:
The order in which the files are loaded also comes into play here - scenes.jl is loaded before shorthands.jl whereas makielayout is loaded after it, IIRC.

Reply:
Ah! Makes sense, thanks.